pyproject.toml

```
@@ -0,0 +1,114 @@
+[build-system]
+requires = ["setuptools>=78.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "wagtail"
+dynamic = ["version"]
+description = "A Django content management system."
+readme = "README.md"
+license = "BSD-3-Clause"
```

suggestion:

```suggestion
license = "BSD-3-Clause"
license-files = [ "LICENSE" ]
```

ref: https://hugovk.dev/blog/2025/improving-licence-metadata/